### PR TITLE
Add Dropbox Passwords Latest

### PR DESCRIPTION
**Pull request by jonotrujillo**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/dropbox-passwords.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "dropbox-passwords" do
+  version :latest
+  sha256 :no_check
```
Comment (miccal) [inline Casks/dropbox-passwords.rb:3]: Please change these to:
```
version "5.1.4"
sha256 "3e586ee75ff0b846371f37e17a12569a2683df112d3cd7f70931e60d53482c15"
```

File: Casks/dropbox-passwords.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url "https://www.dropbox.com/dropbox-passwords-download/install?platform=mac"
```
Comment (miccal) [inline Casks/dropbox-passwords.rb:5]: Please change this to:
```
url "https://clientupdates.dropboxstatic.com/dbx-releng/dropbox_passwords/mac/DropboxPasswords_#{version}.dmg
```
and please add the following `appcast`:
```
appcast "https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://www.dropbox.com/dropbox-passwords-download/mac/beta"
```

File: Casks/dropbox-passwords.rb (hunk 1 above)
Comment (jonotrujillo) [reply Casks/dropbox-passwords.rb]: Done in 1a2e606

File: Casks/dropbox-passwords.rb (hunk 2 above)
Comment (jonotrujillo) [reply Casks/dropbox-passwords.rb]: Done in 1a2e606

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**